cmd/validate_image_test.go

```diff
@@ -221,13 +228,66 @@ func Test_ValidateErrorCommand(t *testing.T) {
 	assert.Empty(t, out.String())
 }
 
+func Test_FailureImageAccessibility(t *testing.T) {
+	validate := func(ctx context.Context, imageRef, policyConfiguration, publicKey, rekorURL string) (*output.Output, error) {
+		return &output.Output{
+			ImageSignatureCheck: output.VerificationStatus{
+				Passed: false,
+				Result: &conftestOutput.Result{Message: "skipped due to inaccessible image ref"},
+			},
+			ImageAccessibleCheck: output.VerificationStatus{
+				Passed: false,
+				Result: &conftestOutput.Result{Message: "image ref not accessible. HEAD registry/image:tag: unexpected status code 404 Not Found (HEAD responses have no body, use GET for details)"},
```

Comment:
Wait, is this actually making an HTTP request to the host `registry`?

Reply:
No, it's just just validating output if it were to have those results.

Reply:
Doh. Of course, the real validate function is never called, that's what this mock is. Thanks for clarifying.